### DISCUSSION TITLE
Preventing Popovers from repositioning on scroll

### DISCUSSION THREAD

**Discussion by coliff** [twbs/bootstrap#38982] [Q&A]
I've been working on a little CSS inspector JS project which uses Bootstrap popovers to display the CSS attributes of a property. It's generally working pretty well.

- Source code: https://github.com/coliff/popover-css-inspector
- Demo: https://coliff.github.io/popover-css-inspector/

One thing I can't figure out is how to prevent the popovers from repositioning. In my case I'd like them **always** to display to the left of the element and not reposition on scroll. I've read through all [the Popover docs](https://getbootstrap.com/docs/5.3/components/popovers/) and the PopperJS docs too.

I've searched for an answer too, but the only question on [Stack Overflow I found is unanswered](https://stackoverflow.com/questions/75838513/prevent-bootstrap-5-popover-from-repositioning-on-scroll). Can anyone help?

**Comment by julien-deramond** [comment]
Really nice project @coliff 🤗 
I thought it would be possible to do that with the [`flip` modifier](https://popper.js.org/docs/v2/modifiers/flip/) but I haven't managed to make it work... Maybe I've missed something while trying it.

**Reply by coliff** [reply]
thanks for checking on this. Please let me know if you ever find a fix or workaround.

Bootstrap [Dropdowns support the `data-bs-display` option](https://getbootstrap.com/docs/5.3/components/dropdowns/#options:~:text=detectOverflow%20docs.-,display,-string) (with default being dynamic and a static option).

I'd love to see this same option available for Popovers and Tooltips!

**Reply by julien-deramond** [reply]
IDK if it can fit with what you trying to achieve but maybe another approach would be to hide the popup if it overflows the viewport with something like [Detect Overflow util](https://popper.js.org/docs/v2/utils/detect-overflow/) 🤷
